schema-lending.graphql

```diff
@@ -529,11 +598,13 @@ type Market @entity {
   " Liquidation penalty (or the liquidation bonus for liquidators) as a percentage value. It is the penalty/bonus price on the collateral when liquidators purchase it as part of the liquidation of a loan that has passed the liquidation threshold "
   liquidationPenalty: BigDecimal!
 
-  " Token that need to be deposited to take a position in protocol "
-  inputToken: Token!
+  " Address of the price oracle used in this market "
+  priceOracle: Bytes
```

Comment:
We should keep the data type for addresses consistent across the schema, either as Bytes or as String

Reply:
If we change addresses to Bytes, might be a good time to change all IDs to Bytes as well, for performance improvements documented here: https://medium.com/edge-node-engineering/two-simple-subgraph-performance-improvements-a76c6b3e7eac

Reply:
If we want to keep the pattern {transaction count}-{hash}-{logindex} it won't be possible to make that of type Bytes.

A few options:
- We don't use that format (I'm against this)
- Or we just use Bytes on the IDs that we can

Reply:
Removed transaction count prefix per convo w/ suds and opted for Bytes wherever possible